### PR TITLE
Add docs for accessing MongoDB externally and fix cert docs

### PR DESCRIPTION
**Pull request by aliscott**

### CODE REVIEW

File: learn/creating-self-signed-certificates.md [hunk 1]
```diff
@@ -88,14 +88,20 @@ You can use self-signed certificates to help secure communication across your in
 
   ```
   export EXT_HOSTNAME=myservice-myproject.abar.cloud
-  export INT_WILDCARD_HOSTNAME="*.myproject.svc.cluster.local"
+  export INT_WILDCARD_HOSTNAME="*.myservice.myproject.svc.cluster.local"
```
Comment (alikhajeh1) [inline learn/creating-self-signed-certificates.md]: the previous wildcard wasn't covering this?

File: additional-services/mongodb.md [hunk 2]
```diff
@@ -82,6 +82,37 @@ Note: to do this you must [configure MongoDB to use SSL](#configuring-ssl)
...
+First we must enable the MongoDB primary router that was included in the clustered MongoDB template:
+
+```
+oc scale --replicas=1 dc/mongodb-primary-router
+```
+
+Then we create the route for the hostname you created the certificate for:
+
+```
+oc expose svc/mongodb --hostname=<hostname>
```
Comment (alikhajeh1) [inline additional-services/mongodb.md]: How does this route send traffic to the primary-router?

File: additional-services/mongodb.md [hunk 3]
```diff
@@ -82,6 +82,37 @@ Note: to do this you must [configure MongoDB to use SSL](#configuring-ssl)
...
+```
+
+Then run the following to configure the passthrough route:
+
+```
+oc patch route/mongodb -p '{"spec":{"tls":{"termination":"passthrough"}}}'
```
Comment (alikhajeh1) [inline additional-services/mongodb.md]: Can't we use the `oc expose --overrides` option to do this as part of the creation?

File: additional-services/mongodb.md [hunk 4]
```diff
@@ -82,6 +82,37 @@ Note: to do this you must [configure MongoDB to use SSL](#configuring-ssl)
 
 Since MongoDB supports SNI (Server Name Indication) you can expose your MongoDB by creating a passthrough route for it.
 
+#### Using the clustered MongoDB
+
+If you are using the clustered MongoDB we need to use a special MongoDB router to ensure that our MongoDB traffic is always hitting the primary node.
```
Comment (alikhajeh1) [inline additional-services/mongodb.md]: If you are using the clustered MongoDB you need to use a special MongoDB router to ensure that your MongoDB traffic is always hitting the primary node.
(we -> you, our -> your? might be easier to understand)

File: learn/creating-self-signed-certificates.md (hunk 1 above)
Comment (aliscott) [inline learn/creating-self-signed-certificates.md]: Nope, this was a bug.

**Pull request merged**